wordpress-mobile/WordPress-iOS
# Issues/7675 add google framework via plistpre

Refs #7675 
Alternate implementation of #7790

To test:
- add new google client ID values to `~/.wpcom_app_credentials` (see template file for key names)
- ensure the wordspace still builds
- tap the "Login in with Google" button
- login to Google
- ensure the success alert appears

*Discussion:*
The values are stored in the credentials file, and either added to the Plist preprocessor or ruby-generated `ApiCredentials.m` as appropriate.

Needs review: @SergioEstevao
